recipes/zbar/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+                "--with-xv" if self.options.with_xv else "--without-xv",
+                "--with-jpeg" if self.options.with_jpeg else "--without-jpeg",
+                "--enable-pthread" if self.options.enable_pthread else "--disable-pthread",
+                "--enable-shared" if self.options.shared else "--disable-shared",
+                "--enable-static" if not self.options.shared else "--disable-static",
+            ]
+            self._env_build.configure(args=env_args, configure_dir=self._source_subfolder)
+        return self._env_build
+
+    def requirements(self):
+        if self.options.with_jpeg:
+            self.requires("libjpeg/9d")
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Zbar can't be built on Windows")
+        if tools.is_apple_os(self.settings.os) and not self.options.shared:
+            raise ConanInvalidConfiguration("Zbar can't be built static on macOS")
+        if self.options.with_imagemagick:   #TODO add when available
+            self.output.warn("There is no imagemagick package available on Conan (yet). This recipe will use the one present on the system (if available).")
+        if self.options.with_gtk:           #TODO add when available
+            self.output.warn("There is no gtk package available on Conan (yet). This recipe will use the one present on the system (if available).")
+        if self.options.with_qt:            #TODO add when available
+            self.output.warn("There is no qt package available on Conan (yet). This recipe will use the one present on the system (if available).")
+        if self.options.with_xv:            #TODO add when available
+            self.output.warn("There is no Xvideo package available on Conan (yet). This recipe will use the one present on the system (if available).")
+
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        env_build = self._configure_autotools()
+        env_build.make()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        env_build = self._configure_autotools()
+        env_build.install()
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        for dot_la_files in glob.iglob(os.path.join(self.package_folder, "lib", '*.la')):
+            os.remove(dot_la_files)
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Linux" and self.options.enable_pthread:
+            self.cpp_info.system_libs = ["pthread"]
+        if tools.is_apple_os(self.settings.os):
+            self.cpp_info.system_libs = ["iconv"]
```

Comment:
Is this the the `libiconv` we have? (https://conan.io/center/libiconv/1.16) Maybe it would be nice to add it for the next iteration of this recipe 🏗️ 

We can open an issue so we don't forget about it